@here/olp-sdk-dataservice-read/test/unit/VolatileLayerClient.test.ts

```diff
@@ -138,11 +144,61 @@ describe("volatileLayerClient", () => {
         expect(partitions).to.be.equal(mockedPartitions);
     });
 
+    it("Should method getPartitions provide data with additionalFields parameter", async () => {        
+        const mockedPartitions = {
```

Comment:
Please, use getPartitionsStub.calledWith to check if layer client sends a request with correct params, instead of checking the mocked response